packages/components/dialog/src/dialog.stories.ts

```diff
@@ -1,129 +1,121 @@
+    disableCancel: false,
+    title: 'Title'
   },
-  render: ({ title, subtitle, bodyContent, closeButton }) => {
+  render: ({ body, closeButton, disableCancel, footerButtons, headerButtons, title, subtitle, maxWidth }) => {
+    const onClick = (event: Event & { target: HTMLElement }): void => {
+      (event.target.nextElementSibling as Dialog).showModal();
+    };
+
     return html`
+      <style>
+        sl-dialog {
+          ${maxWidth ? `--sl-dialog-max-inline-size: ${maxWidth}` : ''};
+        }
+      </style>
       <sl-button fill="outline" size="md" @click=${onClick}>Show Dialog</sl-button>
-      <sl-dialog ?close-button=${closeButton}>
+      <sl-dialog ?close-button=${closeButton} ?disable-cancel=${disableCancel}>
         <span slot="title">${title}</span>
-        <span slot="subtitle">${subtitle}</span>
-        ${bodyContent}
-        <sl-button slot="actions" fill="ghost" variant="default" sl-dialog-close autofocus>Cancel</sl-button>
-        <sl-button slot="actions" fill="solid" variant="primary" sl-dialog-close>Action</sl-button>
+        ${subtitle ? html`<span slot="subtitle">${subtitle}</span>` : nothing} ${body}
+        ${headerButtons ? headerButtons() : nothing}
+        ${footerButtons
+          ? footerButtons()
+          : html`
+              <sl-button slot="actions" fill="ghost" variant="default" sl-dialog-close autofocus>Cancel</sl-button>
+              <sl-button slot="actions" fill="solid" variant="primary" sl-dialog-close>Action</sl-button>
+            `}
       </sl-dialog>
     `;
   }
-};
+} satisfies Meta<Props>;
 
-export const MoreFooterButtons: StoryObj = {
+export const Basic: Story = {
   args: {
-    reverse: false
-  },
-  render: ({ reverse }) => html`
-    <style>
-      sl-button-bar[reverse] > sl-button:first-child {
-        margin-left: auto;
-      }
+    subtitle: 'Subtitle'
+  }
+};
 
-      sl-button-bar:not([reverse]) > sl-button:first-child {
-        margin-right: auto;
-      }
+export const CloseButton: Story = {};
 
-      @media screen and (max-width: 600px) {
-        sl-button-bar {
-          align-items: stretch;
-          flex-direction: column-reverse;
-          flex-grow: 1;
-          flex-shrink: 0;
-        }
+export const DisableCancel: Story = {
+  args: {
+    body: 'You cannot close me by pressing the Escape key, or clicking the backdrop. This dialog also has no close button. The only way to close it is by clicking one of the action buttons.',
+    closeButton: false,
+    disableCancel: true,
+    maxWidth: '300px',
+    title: 'Disable cancel and close button'
+  }
+};
 
-        sl-button-bar[reverse] > sl-button:first-child {
-          margin-left: inherit;
+export const FooterButtons: Story = {
```

Comment:
Can you please show an option for the `reverse` to footer buttons like it was before?

<img width="1102" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/c1bc39c3-c22b-4403-90f8-1a4231236a4e">

Reply:
Fixed